src/app-layout/visual-refresh-toolbar/toolbar/drawer-triggers.tsx

```diff
@@ -135,12 +136,13 @@ export function DrawerTriggers({
               className={clsx(
                 styles['drawers-trigger'],
                 testutilStyles['drawers-trigger'],
-                splitPanelTestUtilStyles['open-button']
+                testutilStyles['split-panel-trigger'],
+                splitPanelResolvedPosition === 'side' && splitPanelTestUtilStyles['open-button']
```

Comment:
Let's keep this test util class assigned in both cases. 

The collapsed split panel will not use this class. All customers' tests will open split panel using this button in every position